src/components/context/PeopleContext.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Person } from '../../types/Person';
+import { getPeople } from '../../utils/api';
+
+interface PeopleContextType {
+  people: Person[];
+  error: string;
+  isLoading: boolean;
+}
+
+const contextValue: PeopleContextType = {
+  people: [],
+  error: '',
+  isLoading: false,
+};
+
+export const PeopleContext =
+  React.createContext<PeopleContextType>(contextValue);
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const PeopleProvider: React.FC<Props> = ({ children }) => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    const fetchPeople = async () => {
+      try {
+        const fetchedPeople = await getPeople();
+
+        setPeople(fetchedPeople);
+      } catch {
```

Comment:
In the catch block, you should also catch a possible error, because now the error does not look informative, for example, you can do this:

```js

catch(error) { setIsError(`Something went wrong: ${error.message}`)}
```